### ISSUE TITLE
Any plans to update this?

### ISSUE THREAD

**Issue by ghuff07** [4andreas/Drono#3]
I was wondering if you had any plans to keep this updated, or to upload it over on lotrointerface.com?

**Comment by Dromo** [comment]
Not really, LotroCompanion has the functionality to import deed data and mark which caches you found automatically. I would suggest using it. 

Can probably make an update for this if desired, although the Deepscrave map scaling was still not fixed by SSG. 
Also not sure how to put treasures vs caches as there is 6 new maps, putting them on same map with different marker would require some more significant recode, and as mentioned, since LotroCompanion can do it (and much more), i lack motivation to assign time to work on this.
